tools/mesh-generators/README.md

```diff
@@ -8,3 +8,20 @@ python generate_unit_cube.py --mesh  coarse.vtk --resolution 0.2
 ```
 
 This requires the `gmsh` and `meshio` Python packages.
+
+# Unit grid mesh generator
+
+These script can generate a unit grid with given amount of point in dimensions x,y,z.
+Run with output file name and mesh sizes.
+Defaults are `x=10`, `y=10`, `z=1`.
+
+```bash
+# 10x10 2D grid with z=0
```

Comment:
`z=0` is not a valid argument option, right?

Reply:
Good point, this is confusing.
The points of the plane have a z coordinate of 0, hence `z=0`.

```suggestion
# 10x10 2D grid with z coordinate of 0
```

Does this make more sense?